internal/controller/argocd.go

```diff
@@ -46,7 +46,13 @@ func (clctrl *ClusterController) InstallArgoCD() error {
 			}
 		}
 
-		argoCDInstallPath := fmt.Sprintf("github.com:kubefirst/manifests/argocd/cloud?ref=%s", pkg.KubefirstManifestRepoRef)
+		var argoCDInstallPath string
+		if clctrl.CloudProvider == "digitalocean" {
+			argoCDInstallPath = "github.com:konstructio/manifests/argocd/cloud?ref=v1.1.0"
+		} else {
+			argoCDInstallPath = fmt.Sprintf("github.com:kubefirst/manifests/argocd/cloud?ref=%s", pkg.KubefirstManifestRepoRef)
+		}
```

Comment:
Not sure about this so feel free to ignore... How come _just_ DigitalOcean gets the new manifests from @konstructio and not everyone? (genuine curiosity lol)

Reply:
the digital ocean k8s upgrade needs argo v2.12 ,so the new version tag is on the charts that is on Konstruct org, the version upgrade may break for other clouds ;-; ;)